umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10.clj

```diff
@@ -45,7 +46,9 @@
 (defn parse-characteristics
   "Returns a seq of UMM characteristic records from the element's child Characteristics."
   [el]
-  (map parse-characteristic (select el "Characteristics/Characteristic")))
+  (seq (remove nil?
+         (map char-data-type-migration/migrate-data-type
```

Comment:
Combine the two maps into a single one by `comp` the two functions.

Reply:
Thanks. Done.